runtime/descriptor.go

```diff
@@ -98,24 +105,54 @@ func propertySet(f *Frame, desc, inst, value *Object) *BaseException {
 
 // makeStructFieldDescriptor creates a descriptor with a getter that returns
 // the field given by fieldName from t's basis structure.
-func makeStructFieldDescriptor(t *Type, fieldName, propertyName string) *Object {
+func makeStructFieldDescriptor(t *Type, fieldName, propertyName string, fieldType fieldDescriptorType) *Object {
```

Comment:
When fieldType is fieldDesciptorRW, you should check that `field.PkgPath` is empty (i.e. the [field is exported](https://golang.org/pkg/reflect/#StructField) and therefore settable). Otherwise we should logFatal. This will allow us to avoid panicking at run time when we accidentally made an unexported field RW in this library.